haskell/test/SMT.hs

```diff
@@ -748,3 +752,6 @@ applyFA2Parameter ContractCall {..} cs = do
     FA2.Transfer tis -> applyTransfer ccSender cs tis
     FA2.Update_operators ops -> foldl' (applyUpdateOperator ccSender) (Right cs) ops
     _ -> error "Unexpected param"
+
+vectorOf :: MonadGen m => Int -> m x -> m [x]
+vectorOf n = Gen.list (Range.singleton n)
```

Comment:
Would it be better to use `Range.linear` or  `Range.constant`, so that we get shrinked values when a test fails?

Reply:
I am not sure, but AFAIU this behavior is equivalent to what we had in QuickCheck version, because AFAIU `vectorOf` doesn't shrink the size and doesn't depend on the size parameter. However, it seems safe to shrink here, so I guess I'll change it to `linear` in a separate commit.